spec/features/registration_spec.rb

```diff
@@ -0,0 +1,86 @@
+feature 'registration' do
+  let(:email)     { Faker::Internet.unique.email }
+  let(:password1) { '1234567890' }
+  let(:password2) { '987654321' }
+
+  context 'signup form' do
+    before { visit '/' }
+
+    scenario 'User registration' do
+      signup(email, password1, password1)
+
+      expect(page).to have_css('.alert')
+      logout
```

Comment:
You should create after block (below all let) for logout and delete it from the test
None of your tests should be connected with the previous test